src/u/setup/index.php

```diff
@@ -0,0 +1,273 @@
+<?php
+if (isset($_POST['save'])) {
+
+    $settings = new \stdClass;
```

Comment:
Probably better as an associative array

Reply:
the obj, is just to make it not throw an error.
it outputs as an keyed array, that generated the json file.